src/index.js

```diff
@@ -1,65 +1,16 @@
-import firebase from 'firebase/compat/app';
-import * as firebaseui from 'firebaseui'
-import 'firebaseui/dist/firebaseui.css'
+import Vue from "vue";
 
-// Your web app's Firebase configuration
-const firebaseConfig = {
-  apiKey: "AIzaSyB3_ktYCDAvZR8ViEgmkKT8CDMPm1Q53Zo",
-  authDomain: "safetydungeon.firebaseapp.com",
-  projectId: "safetydungeon",
-  storageBucket: "safetydungeon.appspot.com",
-  messagingSenderId: "156587932337",
-  appId: "1:156587932337:web:116ae943f4f2ff92790090"
-};
+import "firebaseui/dist/firebaseui.css";
```

Comment:
imports the firebaseui stylesheet

not sure why the path has "/dist" in it

Reply:
Because that's where the authors of the firebasui module put the stylesheet.